libs/ui/packages/native/src/components/Layout/Divider/index.tsx

```diff
@@ -1,10 +1,25 @@
+import React from "react";
 import Flex, { FlexBoxProps } from "../Flex";
 import styled from "styled-components/native";
+import { Text } from "../../..";
 
-const Divider = styled(Flex).attrs<FlexBoxProps>((p: FlexBoxProps) => ({
+const DividerBase = styled(Flex).attrs<FlexBoxProps>((p: FlexBoxProps) => ({
   my: p.my || 4,
   height: 1,
   backgroundColor: "neutral.c40",
 }))``;
 
+const Divider: React.FC<FlexBoxProps & { text?: string }> = (props) => {
+  if (!props.text) return <DividerBase {...props} />;
+  return (
+    <Flex {...props} flexDirection="row" alignItems="center">
+      <DividerBase flex={1} />
+      <Text variant="bodyLineHeight" color="neutral.c60" mx={6}>
+        {props.text}
+      </Text>
+      <DividerBase flex={1} />
```

Comment:
Now the `Divider` has at minimum twice the height than the previous version ? (because there are 2 `DividerBase`) Is it ok for components that already use it ?

Reply:
Existing uses don't use the `text` prop so they fall into this case:
```tsx
if (!props.text) return <DividerBase {...props} />;
```

And regarding the height, it's not actually doubled, because these components are layed out horizontally on a same row. The height might actually be slightly different in that case, if the text box is higher than DividerBase, but it's using values from Figma so it looks like what the designers planned.

Reply:
Ok !